src/components/Buttons/ClickableArea.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2018 - 2022 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { FC } from 'react'
+import styled from 'styled-components'
+
+interface ClickableAreaProps {
+  className?: string
+  onClick?: () => void
+}
+
+const ClickableArea: FC<ClickableAreaProps> = ({ className, onClick, children }) => (
+  <div className={className} role="button" tabIndex={0} onClick={onClick} onKeyPress={onClick}>
+    {children}
+  </div>
+)
+
+export default styled(ClickableArea)`
+  cursor: pointer;
+`
```

Comment:
Since this higher order component does more than listen for click events it would make sense to rename it to something which reflects what it does more closely... Considering it handles multiple inputs, it could be called `InputArea`?